web/src/views/FileBrowserView/FileBrowser.vue

```diff
@@ -55,6 +68,8 @@ import {
   getLocationFromRoute,
 } from '@/utils';
 
+import { girderRest } from '@/rest';
```

Comment:
In this branch `rest.js` exports this by default:
```suggestion
import girderRest from '@/rest';
```